### ISSUE TITLE
Support for public API

### ISSUE THREAD

**Issue by jaller94** [vanita5/mastodon-api#7]
As mentioned in Mastodon’s documentation some API endpoints do not require an authentication. (e.g. `statuses/:id`)

https://github.com/tootsuite/documentation/blob/master/Using-the-API/API.md#timelines

If I try to create a Mastodon object without a an access_token I get the following error:
```
./node_modules/mastodon-api/lib/mastodon.js:345
                    throw new Error('Mastodon config must include \'' + reqKey + '\' when using \'user_auth\'');
                    ^

Error: Mastodon config must include 'access_token' when using 'user_auth'
```

Is it possible to make unauthorized API calls?

**Comment by ida** [comment]
Yes, just pass a dummy-string as access_token: `new Mastodon({access_token: 'dummy'})`

**Comment by jaller94** [comment]
Ok, in this case I see this issue being done when this note gets added to the documentation (readme).

**Comment by eliah-w** [comment]
I will add `publicOnly` as a config parameter in the next version, which is cleaner than a dummy token.
